avocado/utils/pci.py

```diff
@@ -97,20 +98,25 @@ def get_disks_in_pci_address(pci_address):
     return disk_list
 
 
-def get_nics_in_pci_address(pci_address):
+def get_interface_in_pci_address(pci_address, pci_class):
     """
-    Gets network interface(nic) in a PCI address.
+    Gets interface in a PCI address.
 
     :param pci_address: Any segment of a PCI address (1f, 0000:00:1f, ...)
 
-    :return: list of network interfaces in a PCI address.
+    :param clas: Adapter type (FC(fc_host), FCoE(net), NIC(net), SCSI(scsi)..)
+
```

Comment:
I'd suggest adding one or two examples including returned value to make the usage clearer.